homeassistant/components/cloud/assist_pipeline.py

```diff
@@ -1,31 +1,48 @@
 """Handle Cloud assist pipelines."""
+import asyncio
+
 from homeassistant.components.assist_pipeline import (
     async_create_default_pipeline,
     async_get_pipelines,
     async_setup_pipeline_store,
+    async_update_pipeline,
 )
 from homeassistant.components.conversation import HOME_ASSISTANT_AGENT
+from homeassistant.components.stt import DOMAIN as STT_DOMAIN
+from homeassistant.const import Platform
 from homeassistant.core import HomeAssistant
+import homeassistant.helpers.entity_registry as er
 
-from .const import DOMAIN
+from .const import DATA_PLATFORMS_SETUP, DOMAIN, STT_ENTITY_UNIQUE_ID
 
 
 async def async_create_cloud_pipeline(hass: HomeAssistant) -> str | None:
     """Create a cloud assist pipeline."""
+    # Set up stt and tts platforms before creating the pipeline.
```

Comment:
This comment is a tiny bit misleading, we don't setup here, we wait for setup to finish

Reply:
```suggestion
    # Wait for stt and tts platforms to set up  before creating the pipeline.
```